Makefile

```diff
@@ -73,6 +75,16 @@ test: rtf lint
 install: rtf
 	cp -a $^ $(PREFIX)/bin/
 
+.PHONY: docker-image
+docker-image: $(DEPS) Dockerfile
+	docker build --build-arg LDFLAGS="$(LDFLAGS)" -t linuxkit/rtf:$(GIT_COMMIT) .
+
+.PHONY: push
+push: docker-image
+	docker tag linuxkit/rtf:$(GIT_COMMIT) linuxkt/rtf:latest
+	docker push linuxkit/rtf:$(GIT_COMMIT)
+	docker push linuxkit/rtf:latest
```

Comment:
It is kind of weird pushing this to the `linuxkit` org, as we use that for parts of linuxkit not tools. Not sure I can think of a better place though

Reply:
Well, I was thinking about this a little bit... It could also be used a a linuxkit package on `init` if I were to move the binaries over to `/bin/rtf-OS-ARCH`

Reply:
I agree that pushing to the `linuxkit` org is a bit odd. I would prefer if we keep the `linuxkit` repository for core LinuxKit components. Also, this breaks other conventions on the `linuxkit` org, such as no DCT, `latest` tag etc.

maybe push it to `linuxkitprojects` for now?